src/Map/src/Provider/GoogleMaps/GoogleMapsProvider.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\Provider\GoogleMaps;
+
+use Symfony\UX\Map\MapOptionsInterface;
+use Symfony\UX\Map\Provider\ProviderInterface;
+use Symfony\UX\Map\Provider\ProviderTrait;
+
+/**
+ * @author Hugo Alliaume <hugo@alliau.me>
+ */
+final readonly class GoogleMapsProvider implements ProviderInterface
+{
+    use ProviderTrait;
+
+    public static function getDefaultMapOptions(): MapOptionsInterface
+    {
+        return new GoogleMapsOptions();
+    }
```

Comment:
I think we don't need this

Reply:
Hmm so the "provider" instances are some kind of  "builder"  / "factory" right ? (don't mind the name, just on a more macro level)

Reply:
Mmmh more or less, ATM a provider does:
- define its Stimulus `data-controller` attribute
- handle options defined in the DSN (`getOptions()` and `__toString()`) 
- instanciate its `MapOptionsInterface`'s implementation

Reply:
Fixed